src/revpi-factory-reset

```diff
@@ -54,38 +64,34 @@ if [ "$last_part_size" -lt "$last_part_max" ] ; then
 	/sbin/resize2fs "/dev/$last_part"
 fi
 
-if /bin/grep -q "^dtoverlay=kunbus" /boot/config.txt ; then
-	kernel=44
-else
-	kernel=49
-fi
-
-if [ "$kernel" = 49 ] ; then
-	/bin/sed --follow-symlinks -r -i -e "/^dtoverlay=revpi-($full_devicetype_list)/d" /boot/config.txt
+/bin/sed --follow-symlinks -r -i -e "/^dtoverlay=revpi-($full_devicetype_list)/d" /boot/config.txt
+if ! "$with_hat_eeprom"; then
 	echo "dtoverlay=revpi-$ovl" >> /boot/config.txt
 	/usr/bin/dtoverlay "revpi-$ovl"
 	/bin/cp "/boot/overlays/revpi-$ovl-dt-blob.dtbo" /boot/dt-blob.bin 2>/dev/null
 fi
 
 piserial="/usr/sbin/piSerial"
-if [ "$ovl" == flat ]; then
-	#wait for dtoverlay revpi-flat, and tpm modules get probed
-	printf 'waiting for the tpm chip getting ready'
-	wait_time=60
-	for (( i=1; i<=wait_time; i++ ))
-	do
-		if [ "$i" = "$wait_time" ]; then
-			echo "timeout!"
-			echo "The factory reset is stopped here!"
-			exit 1
-		fi
-		if [ -c /dev/tpm0 ]; then
-			echo " ok"
-			break
-		fi
-		printf '%c' '.'
-		sleep 1
-	done
+if ! "$with_hat_eeprom"; then
+	if [ "$ovl" == flat ]; then
+		#wait for dtoverlay revpi-flat, and tpm modules get probed
```

Comment:
Please add a space after the `#`. Also I would suggest to change the comment to something like this
`Wait for tpm chip to become ready. Fail if the chip is not available after $wait_time seconds`

Reply:
is the suggestion regarding to prompt information?
As 
`		# wait for dtoverlay revpi-flat, and tpm modules get probed
		wait_time=60
		printf "Wait for tpm chip to become ready. Fail if the chip is not available after $wait_time seconds"`